src/h5web/visualizations/line/LineVis.tsx

```diff
@@ -18,14 +19,24 @@ function LineVis(): JSX.Element {
     shallow
   );
 
-  const axisDomains = useAxisDomains();
+  const dataDomain = useDataDomain();
```

Comment:
I refactored this hook to return only the data domain. The axis domains are now extended passed inline in the JSX. I think this is more readable with the new config, since we no longer have a prop that takes an `AxisDomains`.